server/logging.properties

```diff
@@ -0,0 +1,65 @@
+###############################################################################
```

Comment:
It was a big surprise for me that the application doesn't have any logging configured yet. I think to add some logging is definitely a good idea, but I am not sure having a `SnapshotHandler` as always on.

Reply:
The snapshot handler is only on if we use this logging.properties.

There are techniques to programatically activate/deactivate it though. I did not add one like an SSH command. The catch is when you're observing a slowdown, you already need monitoring to be on. 

Also I should update the copyright header. (happy 2024!)